docs/getting-started.md

```diff
@@ -111,22 +111,30 @@ From a PowerShell session run the following commands:
 
 ```PowerShell
 # Download and extract desired containerd Windows binaries
-$Version="1.6.4"
+$Version="1.7.0" # update to your preferred version
 curl.exe -L https://github.com/containerd/containerd/releases/download/v$Version/containerd-$Version-windows-amd64.tar.gz -o containerd-windows-amd64.tar.gz
 tar.exe xvf .\containerd-windows-amd64.tar.gz
 
+# If containerd previously installed run:
+Stop-Service containerd
+
 # Copy and configure
-Copy-Item -Path ".\bin\*" -Destination "$Env:ProgramFiles\containerd" -Recurse -Force
-cd $Env:ProgramFiles\containerd\
-.\containerd.exe config default | Out-File config.toml -Encoding ascii
+Copy-Item -Path .\bin\* -Destination (New-Item -Type Directory $Env:ProgramFiles\containerd -Force) -Recurse -Force
+& $Env:ProgramFiles\containerd\containerd.exe config default | Out-File $Env:ProgramFiles\containerd\config.toml -Encoding ascii
```

Comment:
nit: should we just add $Env:ProgramFiles\containerd to the path here? That could make this line and the next few shorter because we wouldn't need to qualify containerd.exe with a path.